types/pubnub/index.d.ts

```diff
@@ -934,6 +939,61 @@ declare namespace Pubnub {
         timetoken: number;
     }
 
+    // APNS2
+    interface APNS2Configuration {
+        collapseId?: string;
+        expirationDate?: Date;
+        targets: APNS2Target[];
+    }
+
+    interface APNS2Target {
+        topic: string;
+        environment?: 'development' | 'production';
+        excludedDevices?: string[];
+    }
+    // NotificationPayloads
+
+    interface BaseNotificationPayload {
+        _subtitle?: string;
```

Comment:
@mohitpubnub 
Any reason for those leading underscores on the properties? I had a quick look at the API Docs, doesn't seem to be the case. https://www.pubnub.com/docs/web-javascript/api-reference-misc#pnnotificationspayload

Also, seems to be missing `buildPayload` method https://www.pubnub.com/docs/web-javascript/api-reference-misc#notificationspayload-arg-3

Reply:
Thank You for drawing my attention to that mistake. For typings declaration, I should not put leading underscores on the properties.
buildPayload(platforms: string[]): object  is missed from my end.

So I'll correct this and raise a new request.